src/NodaTime/CalendarSystem.cs

```diff
@@ -232,10 +236,31 @@ public static CalendarSystem ForId(string id)
         /// <returns>A Hebrew calendar system for the given month numbering.</returns>
         public static CalendarSystem GetHebrewCalendar(HebrewMonthNumbering monthNumbering)
         {
-            Preconditions.CheckArgumentRange(nameof(monthNumbering), (int) monthNumbering, 1, 2);
-            return HebrewCalendars.ByMonthNumbering[((int) monthNumbering) - 1];
+            Preconditions.CheckArgumentRange(nameof(monthNumbering), (int)monthNumbering, 1, 2);
```

Comment:
Touching files should be fine, so long as you don't reformat them.

One option is that when you commit your code in git, use a UI that lets you just commit specific hunks, which should only be the ones with new code. Hopefully only the first commit will need to touch existing code though - the rest is likely to be just the new calculator and tests.

In terms of Visual Studio settings, I suspect my only non-default one is "Insert space after cast" in Formatting/Spacing.